tests/ImageSharp.Tests/Formats/Gif/GifEncoderTests.cs

```diff
@@ -25,6 +26,23 @@ public class GifEncoderTests
             { TestImages.Gif.Ratio4x1, 4, 1, PixelResolutionUnit.AspectRatio }
         };
 
+        [Theory]
+        [WithFile(TestImages.Bmp.Car, PixelTypes.Rgba32)]
+        public void EncodeAllocationCheck<TPixel>(TestImageProvider<TPixel> provider)
```

Comment:
I usually prefer to put these under `ImageSharp.Tests/ProfilingBenchmarks`, and keep them skipped.

Reply:
I didn't actually mean to check that in!